Server/Test_Plan_Files_Antivirus.md

```diff
@@ -136,6 +136,24 @@ The amount of chunks might vary depending on the split size and the original fil
 curl --user $user:$pwd http://$host/$path/remote.php/webdav/virus-chunking-1234-2-0 --request PUT --data-binary "@eicar.com_00" -H "OC-Chunked:1"
 curl --user $user:$pwd http://$host/$path/remote.php/webdav/virus-chunking-1234-2-1 --request PUT --data-binary "@eicar.com_01" -H "OC-Chunked:1"
 ```
+or to do all 3 uploads in a loop:
+```
+BASEFILES="eicar.com eicar_com.zip eicarcom2.zip"
+for BASEFILE in $BASEFILES
+do
+    ID=$RANDOM
+    NUM=0
+    FILES=`ls ${BASEFILE}_*`
+    COUNT=`echo $FILES | wc -w`
+    echo uploading $FILES
+    for FILE in $FILES
+    do
+        echo chunk name: VIRUS$BASEFILE-chunking-${ID}-${COUNT}-${NUM}
+        curl --user $user:$pwd http://$host/$path/remote.php/webdav/VIRUS$BASEFILE-chunking-${ID}-${COUNT}-${NUM} --request PUT --data-binary "@$FILE" -H "OC-Chunked:1"
```

Comment:
You use "VIRUS" In uppercase here at the start of the chunking file name. The manual method uses "virus" in lowercase. Do we care?

Reply:
I don't think we care, its just a random name